BridgeBidder/LCStandard/OpenBid2.cs

```diff
@@ -1,10 +1,13 @@
 using System;
 using System.Collections.Generic;
+using System.Data.SqlTypes;
 
 namespace BridgeBidding
 {
     public class OpenBid2: Open
 	{
+
+		public static StaticConstraint NewSuit = Not(new HasShownSuit(null, true));
```

Comment:
This might be worth its own constraint so we can show `new suit` in the description.

Reply:
Made a new constaint